### PR TITLE
Mock all versions of the OAS cards

### PR DESCRIPTION
**Pull request by TishamIslam**
### Description
Adds all the versions of the OAS cards when a userid of `"default"` is entered

List of proposed changes:

- Changed OAS api to return an array and map it on the dashboard
- Added mapping codes for OAS
- Added a `ProgramTypeCode` for OAS `"OASBeneficial"`
- Added `MapOASCard()` to `BenefitsMapping.js`
- Added Mock data for oas
- `userid` of `"default"` won't throw an error anymore for `cppd.js` api

### CODE REVIEW

File: pages/api/programData/cppd.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ export default async function handler(req, res) {
     //get data
     let userData
     const userid = getCookie('userid', { req, res })
-    if (userid) {
+    if (userid != 'default') {
```
Comment (s-laugh) [inline pages/api/programData/cppd.js:10]: this should get reverted before merging

File: pages/api/programData/oas.js [hunk 2]
```diff
@@ -1,19 +1,42 @@
...
+        userData.forEach((result) => {
+          benefits.push(MapOASCard(result))
+        })
+      } else {
+        //handle the other situations here
+      }
+      res.status(200).json(benefits)
+    } else {
+      //Otherwise send this mock
+      res.status(200).json([
```
Comment (s-laugh) [inline pages/api/programData/oas.js:25]: I don't see a need for us to keep this mock anymore, am I missing something?

**Review by s-laugh — CHANGES_REQUESTED** [review]
Great work! Just some cleanup needed.

File: pages/api/programData/oas.js (hunk 2 above)
Comment (TishamIslam) [reply pages/api/programData/oas.js:25]: Ah that's true

**Review by s-laugh — APPROVED** [review]

**Pull request merged**